python/sglang/srt/managers/scheduler_profiler_mixin.py

```diff
@@ -95,9 +95,11 @@ def init_profile(
         return ProfileReqOutput(success=True, message="Succeeded")
 
     def start_profile(
-        self, stage: Optional[ForwardMode] = None
+        self, stage: Optional[Union[ForwardMode, bool]] = None
```

Comment:
we should fix the caller `self.start_profile(True)` instead of making this function vague